packages/node_modules/pouchdb-checkpointer/src/index.js

```diff
@@ -117,6 +117,10 @@ class CheckpointerInternal {
   getCheckpoint() {
     var self = this;
 
+    if (self.opts.writeSourceCheckpoint === false && self.opts.writeTargetCheckpoint === false) {
```

Comment:
Below the code just uses truthyness so this could be changed to...

```suggestion
    if (!self.opts.writeSourceCheckpoint && !self.opts.writeTargetCheckpoint) {
```

Reply:
Hi Gareth, this was also my first thought.

I've tried it and the result is that the test `checkpointer handles error codes` of `tests/component/test.read_only_replication.js` breaks.
The reason is: it calls `new Checkpointer` without `opts` parameter; resulting in an empty `opts` (line 81) and therefore both options are `undefined` and `!undefined` evaluates to `true`.

So I came to the conclusion, that without specifying the `opts` parameter, the current behaviour must be kept (otherwise, the test might be wrong).

Reply:
Or the fallback/default is wrong

Reply:
Makes sense. Thanks.

@SourceR85 Can I add you as a contributor to the pouchdb org? It might unblock you a bit?

Reply:
> @SourceR85 Can I add you as a contributor to the pouchdb org? It might unblock you a bit?
You're welcome.

Reply:
> @SourceR85 Can I add you as a contributor to the pouchdb org? It might unblock you a bit?

Did you mean, that you want to invite/suggest me as a member to the GitHub group PouchDB?
I'd be delighted.

Reply:
If you merge #8871, we can use the shorter syntax 😉